frontend/src/components/DebugInformation/DebugInformation.tsx

```diff
@@ -56,13 +54,46 @@ const copyToClipboardOptions = {
   message: 'We failed to automatically copy the text, please use: #{key}, Enter\t'
 };
 
+// Will be shown in Kiali Config and hidden in Additional state
+const propsToShow = ["accesibleNamespaces", "authStrategy", "clusters", "gatewayAPIEnabled",
+  "istioConfigMap", "istioIdentityDomain", "istioNamespace", "istioStatusEnabled", "logLevel",
+  "istioCanaryRevision", "istioInjectionAction"];
+
+const tabName = 'tab';
+const defaultTab = 'kialiConfig';
+
+const tabIndex: { [tab: string]: number } = {
+  kialiConfig: 0,
+  additionalState: 1
+};
+
 export class DebugInformation extends React.PureComponent<DebugInformationProps, DebugInformationState> {
-  private textareaRef;
+  aceEditorRef: React.RefObject<AceEditor>;
+  kialiConfig = {};
 
   constructor(props: DebugInformationProps) {
     super(props);
-    this.textareaRef = React.createRef();
-    this.state = { show: false, copyStatus: CopyStatus.NOT_COPIED };
+    this.aceEditorRef = React.createRef();
+
+    for (const key in serverConfig) {
+      if (propsToShow.includes(key)) {
+          if (typeof serverConfig[key] === "string") {
+            this.kialiConfig[key] = serverConfig[key]
+          } else {
+            this.kialiConfig[key] = JSON.stringify(serverConfig[key])
+          }
+      }
+    }
+    // Order config items
+    this.kialiConfig = Object.keys(this.kialiConfig).sort().reduce(
+      (obj, key) => {
+        obj[key] = this.kialiConfig[key];
+        return obj;
+      },
+      {}
+    );
+
+    this.state = { show: false, copyStatus: CopyStatus.NOT_COPIED, currentTab: activeTab(tabName, defaultTab), config: this.kialiConfig };
```

Comment:
I don't really think we need the selected tab to be bookmarkable in the URL.  I think we can simplify and each time you open the Debug Info it should just revert to the "Kiali Config" tab.